eZ/Publish/Core/Persistence/Legacy/Content/Location/Gateway/DoctrineDatabase.php

```diff
@@ -1561,4 +1560,60 @@ private function getAllLocationsQueryBuilder(array $columns)
 
         return $query;
     }
+
+    /**
+     * Create QueryBuilder for selecting node data.
+     *
+     * @param array|null $translations Filters on language mask of content if provided.
+     * @param bool $useAlwaysAvailable Respect always available flag on content when filtering on $translations.
+     *
+     * @return QueryBuilder
+     */
+    private function createNodeQueryBuilder(array $translations = null, bool $useAlwaysAvailable = true): QueryBuilder
+    {
+        $queryBuilder = $this->connection->createQueryBuilder();
```

Comment:
Needed for the change request for the consecutive lines of code
```suggestion
        $dbPlatform = $this->connection->getDatabasePlatform();
        $queryBuilder = $this->connection->createQueryBuilder();
```